src/main/resources/db/changesets/20231129_snapshotrequests.yaml

```diff
@@ -0,0 +1,60 @@
+databaseChangeLog:
+  - changeSet:
+      id: snapshot_requests
+      author: rjohanek
+      changes:
+        - createTable:
+            tableName: snapshot_requests
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

Comment:
It doesn't hurt to do this (and it looks like we usually do it) but I'm pretty sure this has no effect since a primary key must also be non-null. https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS